index.html

```diff
@@ -1,9 +1,21 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" href="./src/assets/images/webLogo.png">
+    <link rel="icon" href="./src/assets/images/webLogo.png" />
```

Comment:
Removing and committing the same thing ?

Reply:
Isn't it standard to close self closing link tags with "/>"?